cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -13,34 +13,137 @@
 # limitations under the License.
 """Commands for sketches."""
 
+import json
 import click
+import pandas as pd
 
 
 @click.group("sketch")
 def sketch_group():
     """Manage sketch."""
 
 
-@sketch_group.command("list")
+@sketch_group.command("list", help="List all sketches. [text,json]")
 @click.pass_context
 def list_sketches(ctx):
     """List all sketches."""
     api_client = ctx.obj.api
+    output = ctx.obj.output_format
+    d = []
+
     for sketch in api_client.list_sketches():
-        click.echo(f"{sketch.id} {sketch.name}")
+        d.append({"id": sketch.id, "name": sketch.name})
+
+    sketch_panda = pd.DataFrame(d, columns=["id", "name"])
+    if output == "json":
+        click.echo(sketch_panda.to_json(orient="records", indent=4))
+    elif output == "text":
+        click.echo(f"{sketch_panda.to_string(index=False)}")
+    else:
+        click.echo(f"Output format {output} not implemented.")
+        ctx.exit(1)
 
 
-@sketch_group.command("describe")
+@sketch_group.command(
+    "describe",
+    help="Describe the active sketch [text,json]",
```

Comment:
Let's not add the supported formats here (hard coded). It will only get outdated in the future.